### PR TITLE
module: fix NODE_MODULE environment works

### PR DESCRIPTION
**Pull request by yorkie**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added

The `module` uses the `process.env.NODE_ENV` before it gets setup, this just moves the function `setupEnviron()` in head of the bootstrap file.

### CODE REVIEW

**Comment by yorkie** [comment]
This also provides a way to set NODE_PATH in our testsets :)

File: tools/testrunner.py [hunk 1]
```diff
@@ -233,12 +235,20 @@ def run_test(self, testfile, timeout):
 
             command = ["valgrind"] + valgrind_options + command
 
+        myenv = os.environ.copy()
```
Comment (legendecas) [inline tools/testrunner.py]: name `case_environs` may be better?

File: test/run_pass/test_module_require2.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var assert = require('assert');
+var dir = process.cwd() + '/run_pass/require2';
```
[the remark this answers is not in the document]
Comment (yorkie) [reply test/run_pass/test_module_require2.js]: Thanks, I will make remove 👍

**Comment by yorkie** [comment]
Merged, thanks for your reviews :)

**Pull request merged**